src/devices/FtCommon/FtDevice.cs

```diff
@@ -2,9 +2,11 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
+using System.Device.Gpio;
 using System.Device.I2c;
+using System.Device.Spi;
 
-namespace Iot.Device.Ft4222
+namespace Iot.Device.FtCommon
 {
     /// <summary>
     /// FT4222 device information
```

Comment:
This class should now derive from `Board`. Many of the methods already match, I think.

Reply:
I have checked and was not fully sure but will definitely try.